### PR TITLE
Enhancement/sigfixes

### PR DESCRIPTION
**Pull request by ric-jon**
Some more SIG fixes

### CODE REVIEW

File: src/main/java/application/controllers/MainController.java [hunk 1]
```diff
@@ -172,7 +172,7 @@ public boolean isMetaDataLoaded() {
 
     /**
      * Method to set whether the MetaData is loaded or not
-     * @param x
+     * @param x boolean
```
Comment (ghost) [inline src/main/java/application/controllers/MainController.java:175]: "boolean" is not really giving a good description of what x is used for.

Comment (dvantetering) [inline src/main/java/application/controllers/MainController.java:175]: What else is there to say about a boolean?

File: src/main/java/application/fxobjects/layout/GraphLayout.java [hunk 2]
```diff
@@ -129,47 +127,29 @@ public void execute() {
...
+                    double yCoord = currentY - evenOffset - (child.getCellShape().getLayoutBounds().getHeight() / 2);
+                    if (child.isRelocatedY()) { yCoord = child.getLayoutY(); }
+                    if (yCoord > maxHeight) { maxHeight = yCoord; }
+                    child.relocate(currentX - (child.getCellShape().getLayoutBounds().getWidth() / 2), yCoord);
+                    evenOffset = (yOffset / 2) * modifier;
                     modifier *= -1;
-                    if (modifier > 0) {
-                        modifier++;
-                    }
+                    if (modifier > 0) { modifier++; }
```
Comment (ghost) [inline src/main/java/application/fxobjects/layout/GraphLayout.java:144]: This if-statements should not be placed on only one line

File: src/main/java/application/fxobjects/layout/GraphLayout.java [hunk 3]
```diff
@@ -129,47 +127,29 @@ public void execute() {
...
+                    double yCoord = currentY + oddOffset - (child.getCellShape().getLayoutBounds().getHeight() / 2);
+                    if (child.isRelocatedY()) { yCoord = child.getLayoutY(); }
+                    if (yCoord > maxHeight) { maxHeight = yCoord; }
+                    child.relocate(currentX - (child.getCellShape().getLayoutBounds().getWidth() / 2), yCoord);
+                    oddOffset = yOffset * modifier;
                     modifier *= -1;
-                    if (modifier < 0) {
-                        modifier--;
-                    }
+                    if (modifier < 0) { modifier--; }
```
Comment (ghost) [inline src/main/java/application/fxobjects/layout/GraphLayout.java:152]: This if-statement should just span three lines

**Comment by dvantetering** [comment]
No further remarks

**Pull request merged**